software/control/widgets.py

```diff
@@ -1110,11 +1112,19 @@ def set_temperature(self):
     def update_measured_temperature(self, temperature):
         self.label_temperature_measured.setNum(temperature)
 
-    def change_full_res(self, index):
-        res_strings = self.dropdown_res.currentText().split("x")
-        res_x = int(res_strings[0])
-        res_y = int(res_strings[1])
-        self.camera.set_resolution(res_x, res_y)
+    def change_binning(self, binning_text):
+        binning_parts = binning_text.split("x")
+        binning_x = int(binning_parts[0])
+        binning_y = int(binning_parts[1])
+
+        old_binning_x, old_binning_y = self.camera.binning
+
+        self.camera.set_binning(binning_x, binning_y)
+
+        scale_x = old_binning_x / binning_x
+        scale_y = old_binning_y / binning_y
```

Comment:
with `crop_width_no_binning/binning` in approach these additional scale variables won't be needed